.babelrc

```diff
@@ -1,4 +1,10 @@
 {
-  "plugins": ["transform-class-properties"],
+  "plugins": [
+    ["transform-class-properties"],
+    ["import", {
```

Comment:
Is there any way to just include all the Ant CSS the same way we include other CSS from node_modules? It's kind of unexpected that a library would require its own babel plugin.

Reply:
This allows for the most simple approach to partial imports for the css from antd. We can pull in all the css easily but i was trying to avoid that